tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/UserEvents/UserManagerCreateIntegration.cs

```diff
@@ -84,7 +85,11 @@ internal static TReturn OnAsyncMethodEnd<TTarget, TReturn>(TTarget instance, TRe
                 if (security.IsAnonUserTrackingMode)
                 {
                     var anonId = UserEventsCommon.GetAnonId(id);
-                    tryAddTag(Tags.AppSec.EventsUsers.SignUpEvent.SuccessUserId, anonId);
+                    if (anonId != null)
```

Comment:
There's a bunch more of these - feel free to ignore if you don't care about `""`

Reply:
yes, fixed in [8fa918f](https://github.com/DataDog/dd-trace-dotnet/pull/6225/commits/8fa918f63e234e07613f17ed1634f12fdb342a0e)